### ISSUE TITLE
getQuote Yahoo Error 404

### ISSUE THREAD

**Issue by craigp67** [joshuaulrich/quantmod#386]
When using getQuote from yahoo today, I started to get a 404 error (instead of the 401 error) we'd been getting.

[Describe the issue]

I expected to get a df from get Quote but got the error message shown below instead.

df <- getQuote(QuoteSymbols, src = "yahoo", what=yahooQF(QuoteAttr))
Error in open.connection(con, "rb") : HTTP error 404

Am using Ethan's patch i.e. 

packageVersion("quantmod")  
[1] ‘0.4.22.1’

Seems like another user, Giovanni, also had this issue and added to the thread of the getQuote Yahoo Error 401 issue.

I tried going back to the CRAN quantmod version 0.4.22 but that just gave the 401 error again instead.

**Comment by AlessandroManilii** [comment]
Same problem in same conditions reported by @craigp67.

**Comment by tian-y** [comment]
I found a workaround from the thread on the 401 error #382 
Thanks to @theinvestmentbox
```
remotes::install_github("ljupch0/yfinance")
yfinance::get_price()
```

**Comment by joshuaulrich** [comment]
This is a duplicate of #382 and was fixed in #383.
